tests/ArgentPonyWarcraftClient.Tests/GameDataApi/ModifiedCraftingApiTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using ArgentPonyWarcraftClient.Tests.Properties;
+using Xunit;
+
+namespace ArgentPonyWarcraftClient.Tests.GameDataApi
+{
+    public class ModifiedCraftingApiTests
+    {
+        [Fact]
+        public async void GetModifiedCraftingIndexAsync_Gets_ModifiedCraftingIndex()
```

Comment:
Is there a particular reason for using `async void` return type on these test methods? My understanding from [Microsoft's async/await best practices](https://docs.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming#avoid-async-void) is that it's generally preferred to use `async Task`. Granted, the perspective of that document is for application code where someone would be calling the method directly and this is a unit test, so I'm not sure if there is a specific benefit to this in a test.

Reply:
There's no good reason to be using `async void`.  I converted these tests, and I'll do a separate PR to convert the remaining tests.  Thanks for pointing that out.